src/workerd/api/node/zlib-util.h

```diff
@@ -338,8 +338,10 @@ class ZlibUtil final: public jsg::Object {
     }
 
   protected:
-    CompressionContext* context() {
-      return &context_;
+    CompressionContext context;
```

Comment:
This creates a use-after-free error because allocations will be destroyed before context.